lession_13/homework_13.py

```diff
@@ -0,0 +1,24 @@
+
+class Rhombus:
+    def __init__(self, side_a, corner_a):
+        self.set_side_a(side_a)
+        self.set_angle_a(corner_a)
+
+    def set_side_a(self, side_a):
```

Comment:
Such method needs to be private as they are setter for attribute creation AND are used inside constructor method.